### PR TITLE
refactor: rename COSMOTECH_URL to ORGANIZATION_URL

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by ardvi — APPROVED** [review]
Little comment but LGTM :+1:

File: src/config/HelpMenuConfiguration.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "APP_VERSION": null,
-  "COSMOTECH_URL": "https://cosmotech.com",
+  "ORGANIZATION_URL": "https://cosmotech.com",
```
Comment (esasova) [inline src/config/HelpMenuConfiguration.json:3]: We don't need to change it in _HelpMenuConfiguration.vanilla.json_ ? :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
One question but LGTM :+1:

File: src/config/HelpMenuConfiguration.json (hunk 1 above)
Comment (csm-thu) [reply src/config/HelpMenuConfiguration.json:3]: You're right, I'll add this too, thank you ! :smile:

**Pull request merged**